services/apps/alcs/src/alcs/notice-of-intent/notice-of-intent-meeting/notice-of-intent-meeting.controller.ts

```diff
@@ -83,7 +83,7 @@ export class NoticeOfIntentMeetingController {
     fileNumber: string,
     meeting: CreateNoticeOfIntentMeetingDto,
   ) {
-    const noi = await this.noiService.getOrFail(fileNumber);
+    const noi = await this.noiService.getOrFailByUuid(fileNumber);
```

Comment:
a little bit odd to have fileNumber as paramter for getOrFailByUuid but I assume this is a leftover from "application"

Reply:
renamed to uuid as it is uuid